backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/work/impl/CountWork.java

```diff
@@ -87,7 +87,7 @@ protected ElasticsearchRequest buildRequest() {
 
 			if ( timeoutManager.hasHardTimeout() ) {
 				// set timeoutValue and timeoutUnit only for hard timeout
```

Comment:
I think this comment is obsolete. Maybe remove it?

Reply:
Sure

Reply:
I rephrased into:
> // soft timeout has no meaning for a count work

Reply:
It should be more expressive... Or at least I hope so